src/styles/blocks/card.scss

```diff
@@ -0,0 +1,92 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  background: #fff;
+  transition: all 0.3s;
```

Comment:
Don't use transition all, specify to what property it should apply.
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/4ac45442-c416-4674-9387-baf3c1e9ff3b)